### PR TITLE
support VMFive ad

### PR DESCRIPTION
**Pull request by cades**
- implements VMFive ad  integration
- provide 3 ad type examples

### CODE REVIEW

File: ads/vmfive.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```
Comment (lannka) [inline ads/vmfive.js]: 2017

File: ads/vmfive.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+function createAdUnit(win, placementId, adType) {
+  const el = document.createElement('vmfive-ad-unit');
+  el.setAttribute('placement-id', placementId);
+  el.setAttribute('ad-type', adType);
+  win.document.body.appendChild(el);
+}
+
+function setupSDKReadyCallback(win, appKey) {
+  win.onVM5AdSDKReady = function(sdk) {
```
Comment (lannka) [inline ads/vmfive.js]: arrow function is preferred

File: ads/vmfive.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  return new Promise((resolve, reject) => {
+    loadScript(win, 'https://man.vm5apis.com/dist/adn-web-sdk.js', resolve, reject);
+  });
+}
+
+function createAdUnit(win, placementId, adType) {
+  const el = document.createElement('vmfive-ad-unit');
+  el.setAttribute('placement-id', placementId);
+  el.setAttribute('ad-type', adType);
+  win.document.body.appendChild(el);
```
Comment (lannka) [inline ads/vmfive.js]: append to the container, not the body. 
```
win.document.getElementById('c').appendChild(el);
```

Comment (cades) [inline ads/vmfive.js]: fixed

File: ads/vmfive.js (hunk 2 above)
Comment (cades) [inline ads/vmfive.js]: fixed

File: ads/vmfive.js (hunk 1 above)
Comment (cades) [inline ads/vmfive.js]: fixed

**Comment by cades** [comment]
hi @lannka, thanks for reviewing!

I'm sure `renderStart()` is called when the ad is ready. now we're trying to reduce the preparation time.

**Comment by cades** [comment]
@lannka thanks!

**Pull request merged**